dc-infrastructure.tf

```
@@ -9,6 +9,8 @@ module "base-infrastructure" {
   domain           = var.domain
   namespace        = local.namespace
   share_home_size  = "5Gi"
+
+  enable_tcp = local.install_bitbucket
```

```suggestion
  enable_ssh_tcp = local.install_bitbucket
```

I am assuming if there are some other products with TCP requirements they would have a separate flag. Also this means that the settings will need to be updated.